### PR TITLE
Basic querying

### PR DESCRIPTION
**Pull request by ajayns**
This illustrates basic querying, documentation will be added once this is merged

Ref:  #2

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
 import Layout from '../components/layout';
 
-const IndexPage = () => (
+const IndexPage = ({ data }) => (
   <Layout>
-    <h1>Hi people</h1>
-    <p>Welcome to your new Gatsby site.</p>
-    <p>Now go build something great.</p>
-    <Link to="/page-2/">Go to page 2</Link>
+    <h2>{data.ploneDocument.title}</h2>
```
Comment (datakurre) [inline src/pages/index.js]: Semantically this should be H1.

If there is H1 in layout, maybe it should be semantically something else?

**Comment by datakurre** [comment]
Try to also follow HTML5 best practices for semantic elements (header, article, section,back, aside etc...)

**Comment by datakurre** [comment]
https://www.smashingmagazine.com/2011/11/html5-semantics/

**Comment by ajayns** [comment]
@datakurre Thanks, will read up, research and try to follow best practices

File: src/pages/index.js (hunk 1 above)
Comment (ajayns) [reply src/pages/index.js]: H1 was used in the header, hence I decided to go with h2. What would be the alternative here?

Comment (datakurre) [reply src/pages/index.js]: If every page has the same H1, it becomes meaningless.

I believe, nowadays it is ok to have multiple H1 on the same page, if they are wrapped with `<article/>` or similar semantic container.

Simpler alternative is to use semantically meaningless tag (div or span) in header "H1" and only make it look big with CSS.

**Comment by ajayns** [comment]
@datakurre does this look good or are more changes required?

File: src/components/header.js [hunk 2]
```diff
@@ -23,11 +23,20 @@ const Header = ({ siteTitle }) => (
...
       </h1>
+      <h5
+        style={{
+          color: 'white',
+          textDecoration: 'none',
+          margin: '16px 0 0 0',
+        }}
+      >
+        {siteData.subTitle}
+      </h5>
```
Comment (datakurre) [inline src/components/header.js]: h5 after h1?

headers should not be used just for different font sizes

Comment (ajayns) [reply src/components/header.js]: Oh maybe I should do a bit more reading. But the gist of what happening here is you're saying headers should be used only for titles in document type views and not for just visual purposes?

**Pull request merged**